sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2451,17 +2463,28 @@ object Radar {
      * @see [](https://radar.com/documentation/api#send-a-custom-event)
      *
      * @param[name] The name of the conversion.
-     * @param[data] The metadata associated with the conversion.
+     * @param[metadata] The metadata associated with the conversion.
      * @param[callback] A callback.
      */
     @JvmStatic
-    fun logConversion(name: String, data: JSONObject?, callback: RadarLogConversionCallback) {
+    fun logConversion(name: String, metadata: JSONObject?, callback: RadarLogConversionCallback) {
```

Comment:
note: no default params for metadata here. should we add them? and if we do, do we want to generate overloads for them in java? https://kotlinlang.org/docs/java-to-kotlin-interop.html#overloads-generation